xlators/features/shard/src/shard.c

```diff
@@ -1488,15 +1489,16 @@ shard_inode_ctx_update(inode_t *inode, xlator_t *this, dict_t *xdata,
          * set 0.
          */
 
-        shard_inode_ctx_set(inode, this, buf, size, SHARD_MASK_BLOCK_SIZE);
+        __shard_inode_ctx_set(inode, this, buf, size, SHARD_MASK_BLOCK_SIZE);
     }
     /* If the file is sharded, also set the remaining attributes,
      * except for ia_size and ia_blocks.
      */
     if (size) {
```

Comment:
nit: I think we can indent this and get it before line 1493 - size is only going to be (possibly) not zero if the 'if' clause in line 1483 is true

Reply:
done